blog/release-v1.11/index.mdx

```diff
@@ -0,0 +1,138 @@
+---
+authors:
+- KubeEdge SIG Release
+categories:
+- General
+- Announcements
+date: 2023-10-25
+draft: false
+lastmod: 2023-10-25
+summary: KubeEdge v1.11 is live!
+tags:
+- KubeEdge
+- kubeedge
+- edge computing
+- kubernetes edge computing
+- K8s edge orchestration
+- edge computing platform
+- cloud native
+- iot
+- iiot
+- release v1.11
+- v1.11
+title: KubeEdge v1.11 is live!
+---
+
+On October 21, 2023, KubeEdge released v1.11, introducing several exciting new features and enhancements that significantly improve node group management, mapper development, installation experience, and overall stability.
```

Comment:
```suggestion
On Jun 20, 2022, KubeEdge released v1.11, introducing several exciting new features and enhancements that significantly improve node group management, mapper development, installation experience, and overall stability.
```
There seems to be a problem with the version time here. We should write the initial release time instead of the patch version time, right? @Shelley-BaoYue

Reply:
The first .0 version

Reply:
done